.devops/deploy-pipelines.yml

```diff
@@ -64,14 +56,13 @@ variables:
     mavenGoals: 'package'
     settingsXmlsecureFileName: '$(SETTINGS_XML_RO_SECURE_FILE_NAME)'
   ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/master') }}:
-    # ${{ if eq(variables.isPROD, true) }}:
     environment: 'PROD'
     dockerRegistryServiceConnection: '$(PROD_CONTAINER_REGISTRY_SERVICE_CONN)'
     kubernetesServiceConnection: '$(PROD_KUBERNETES_SERVICE_CONN)'
     containerRegistry: '$(PROD_CONTAINER_REGISTRY_NAME)'
     deployment: deployment-prod
     selfHostedAgentPool: $(PROD_AGENT_POOL)
-    mavenGoals: 'deploy'
+    mavenGoals: 'package'
```

Comment:
we can use deploy goal and RW file secret

Reply:
We spoke with @dariopelliccioli-sia and the deploy of the jar is not required. Only the docker image is needed